test/delivery_test.rb

```diff
@@ -22,6 +22,16 @@
       -> { MissingToMailer.deliver }.must_raise Lotus::Mailer::MissingDeliveryDataError
     end
 
+    it "Lotus::Mailer::MissingDeliveryDataError bubles up the original Mail::ArgumentError message" do
+      exception = assert_raises(Lotus::Mailer::MissingDeliveryDataError) do
```

Comment:
Used `exception = assert_raises` to get the exception and check the actual error message later on.Since the test basically needs to know if the errors bubbled up the original message.